app/src/main/java/com/vinh/dictionary_1/data/model/Word.java

```diff
@@ -114,6 +117,14 @@ public void setWordWithoutDiacritic(String wordWithoutDiacritic) {
         mWordWithoutDiacritic = wordWithoutDiacritic;
     }
 
+    public boolean isBookmarked() {
+        return bookmarked;
+    }
+
+    public void setBookmarked(boolean bookmarked) {
+        bookmarked = bookmarked;
```

Comment:
ERROR: [STYLE][DLS_DEAD_LOCAL_STORE_SHADOWS_FIELD] Dead store to bookmarked rather than field with same name in com.vinh.dictionary_1.data.model.Word.setBookmarked(boolean)
http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE_SHADOWS_FIELD